configs/FLYTEX405V2/config.h

```diff
@@ -106,8 +109,8 @@
 #define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_OFF
 #define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
 #define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
-#define DEFAULT_VOLTAGE_METER_SCALE 110
-#define DEFAULT_CURRENT_METER_SCALE 326
+#define DEFAULT_VOLTAGE_METER_SCALE 190
```

Comment:
```suggestion
#define DEFAULT_VOLTAGE_METER_SCALE 210
```
in case 200/10 divider it should be proper value